Dockerfile

```diff
@@ -62,7 +62,7 @@ RUN apt install -y bash
 
 # The following lines are from the mpiFileUtils (nnf-mfu) Dockerfile;
 # do not change them unless you know what it is you are doing
-ARG port=2222
+ARG port=22
```

Comment:
Just delete this, and line 67. No need for the complexity

Reply:
Done.